Cargo.toml

```diff
@@ -14,6 +14,7 @@ categories = ["conda"]
 homepage = "https://github.com/mamba-org/rattler"
 repository = "https://github.com/mamba-org/rattler"
 license = "BSD-3-Clause"
+license-files = { paths = ["LICENSE"] }
```

Comment:
This one is not needed if license has been specified

Reply:
see: https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields

Reply:
i just realised, fixed it